doc/howtos/rdtraining/basicviews.rst

```diff
@@ -0,0 +1,220 @@
+.. _howto/rdtraining/basicviews:
+
+===========
+Basic Views
+===========
+
+We have seen in the :ref:`previous chapter <howto/rdtraining/firstui>` that Odoo was able to
+generate default views for a given model. In practice, the default view is **never** acceptable
+for a business application.
+
+Views are defined in the XML files along with actions and menus. They are entries of the
+``ir.ui.view`` model.
+
+In our real estate module, we need to organize the fields in a logical way:
+
+- in the list (tree) view, we want to display more than the name
+- in the form view, the fields should be grouped
+- in the search view, we must be able to search on more than the name. Moreover, a filter is
+  created to filter on the 'Available' properties and there is a shortcut to group by postcode.
+
+List
+====
+
+**Reference**: the documentation related to this topic can be found in
+:ref:`reference/views/list`.
+
+.. note::
+
+    **Goal**: at the end of this section, the list view should look like this:
+
+    .. image:: basicviews/media/list.png
+      :align: center
+      :alt: List view
+
+List views, also called tree views, display records in a tabular form.
+
+Their root element is ``<tree>``. The simplest form of the tree view simply
+lists all the fields to display in the table (each field as a column):
+
+.. code-block:: xml
+
+    <tree string="Tests">
+        <field name="name"/>
+        <field name="last_seen"/>
+    </tree>
+
+A simple example can be found
+`here <https://github.com/odoo/odoo/blob/6da14a3aadeb3efc40f145f6c11fc33314b2f15e/addons/crm/views/crm_lost_reason_views.xml#L46-L54>`__.
+
+.. exercise:: Add a custom list view
+
+    Define a list view for the ``estate.property`` model in the appropriate XML file. Check the
+    **Goal** of the section for the fields to display.
+
+    Tip: do not add the ``editable="bottom"`` attribute that you can find in the example above.
+    We'll come back to it later.
```

Comment:
Tip: do not forget to change the labels to be more user-friendly for instance the name field can be called 'Title'